packages/client/hmi-client/src/page/data-explorer/DataExplorer.vue

```diff
@@ -2,15 +2,6 @@
 	<main>
 		<section class="flex h-full relative overflow-hidden">
 			<tera-slider-panel content-width="240px" direction="left" header="Filters" v-model:is-open="isSliderFacetsOpen">
-				<template v-slot:content>
-					<tera-facets-panel
-						v-if="viewType === ViewType.LIST"
-						:facets="facets"
-						:filtered-facets="filteredFacets"
-						:result-type="resourceType"
-						:docCount="docCount"
-					/>
-				</template>
 			</tera-slider-panel>
```

Comment:
If this contains nothing now, should the `tera-slider-panel` be deleted or are there still plans to use it?